From 7c18587f8293dbed6707e434c720381ced30ea1d Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Sat, 17 Jun 2017 16:16:58 +0800
Subject: [PATCH 178/767] dpaa_eth: use ptp-timer phandle instead of
 ptimer-handle

Layerscape DPAA platforms have updated dts to use ptp-timer phandle
instead of ptimer-handle for Fman RTC node. This patch is to
update it in driver.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
index e6c47a8a1da4..fbe61da2d141 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
@@ -575,7 +575,7 @@ dpa_mac_probe(struct platform_device *_of_dev)
 	}
 
 #ifdef CONFIG_FSL_DPAA_1588
-	phandle_prop = of_get_property(mac_node, "ptimer-handle", &lenp);
+	phandle_prop = of_get_property(mac_node, "ptp-timer", &lenp);
 	if (phandle_prop && ((mac_dev->phy_if != PHY_INTERFACE_MODE_SGMII) ||
 			((mac_dev->phy_if == PHY_INTERFACE_MODE_SGMII) &&
 			 (mac_dev->speed == SPEED_1000)))) {
@@ -595,7 +595,7 @@ dpa_mac_probe(struct platform_device *_of_dev)
 	if ((mac_dev->phy_if != PHY_INTERFACE_MODE_SGMII) ||
 	    ((mac_dev->phy_if == PHY_INTERFACE_MODE_SGMII) &&
 			 (mac_dev->speed == SPEED_1000))) {
-		ptp_priv.node = of_parse_phandle(mac_node, "ptimer-handle", 0);
+		ptp_priv.node = of_parse_phandle(mac_node, "ptp-timer", 0);
 		if (ptp_priv.node) {
 			ptp_priv.of_dev = of_find_device_by_node(ptp_priv.node);
 			if (unlikely(ptp_priv.of_dev == NULL)) {
-- 
2.17.0

